src/app/pages/authentication/registration/registration.component.ts

```diff
@@ -99,9 +100,10 @@ export class RegistrationComponent implements OnInit, OnDestroy {
       userName: formValue.userName,
       userPassword: this.registrationForm.get('passwords')?.get('userPassword')?.value,
       userEmail: formValue.userEmail,
+      isAdmin: false,
```

Comment:
It would ve better to hide this logic inside AuthService
F.e we could have 2 methods createAdmin and create createUser, where inside createAdmin you add isAdmin: true